### ISSUE TITLE
does not support JSONC

### ISSUE THREAD

**Issue by Zamiell** [knightedcodemonkey/duel#32]
Hello, and thanks for the package.

I tried it out on my repo and got the following error:

```
$ npx duel
The config file found at D:\Repositories\foo\tsconfig.json is not parsable as JSON.
```

The error message implies that `duel` is parsing the "tsconfig.json" file using JSON. But that is a bug, because "tsconfig.json" files are not JSON files, they are JSONC files. JSONC is a format which allows both comments and trailing commas.

You are supposed to parse it with something like the following:

```ts
import JSONC from "jsonc-parser";

const json = JSONC.parse(fileContents);
```

I believe that this is how TypeScript itself parses it, as well as VSCode, and so on.

**Comment by knightedcodemonkey** [comment]
Thanks for the reminder that tsconfig.json is not really JSON.

Not strictly necessary, but if you could provide an example of your configuration that would be helpful.

**Comment by Zamiell** [comment]
Sure thing, the tsconfig.json that I was using is here: https://github.com/IsaacScript/isaacscript/blob/main/packages/isaacscript-common-ts/tsconfig.json

**Comment by knightedcodemonkey** [comment]
You want to give #33 a look over? Specifically, I did not add support for `allowEmptyContent`...

**Comment by Zamiell** [comment]
seems fine, thanks

**Comment by knightedcodemonkey** [comment]
Published to npm [v1.0.4](https://github.com/knightedcodemonkey/duel/releases/tag/v1.0.4).
